tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers_test.ts

```diff
@@ -676,6 +690,8 @@ describe('Debugger graphs reducers', () => {
     const state = createDigestsStateWhileLoadingExecutionDigests(
       pageSize,
       numExecutions,
+      0 /* Begin of loading range. */,
```

Comment:
nit: Prefer passing dict so the code reads better than describing the arg position with comments like this.

Reply:
Done. This creates some inconsistency with other test-helper functions. But I think it's justified in this case because the arg list is indeed getting too long.